adapters/webhdfs-adapter/src/main/java/com/connexta/replication/adapters/webhdfs/WebHdfsNodeAdapter.java

```diff
@@ -382,9 +384,21 @@ String getVersion4Uuid(String fileUrl, Date modificationTime) {
     return addFailedItemsToRelevantFiles(results, oldFiles, failedItemIds);
   }
 
-  private List<FileStatus> addFailedItemsToRelevantFiles(List<FileStatus> relevantFiles, Map<String, FileStatus> oldFiles, List<String> failedItemIds) {
+  /**
+   * Iterates through the list of failed IDs to look up the associated {@link FileStatus} in the
+   * map.
+   *
+   * @param relevantFiles - the list of {@link FileStatus} to be returned
+   * @param oldFiles - the list of {@link FileStatus} with a modification date after the filter date
```

Comment:
`oldFiles` contains files with mod date that occurred before the filter date.

Reply:
Whoops I must be losing my mind making that mistake twice 😅